esi_leap/manager/service.py

```diff
@@ -88,7 +88,7 @@ def _expire_offers(self):
                o.end_time <= timeutils.utcnow():
                 LOG.info("Expiring offer %s for %s %s",
                          o.uuid, o.resource_type, o.resource_uuid)
-                o.expire(self._context)
+                offer.Offer.expire(o.uuid, self._context)
```

Comment:
This looks like it might be incorrect... ?

Reply:
Yep. Thanks for catching that.